src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -868,6 +868,11 @@ internal static CultureData GetCultureData(int culture, bool bUseUserOverride)
 
             if (GlobalizationMode.Invariant)
             {
+                if (!GlobalizationMode.PredefinedCulturesOnly)
```

Comment:
Please try adding some validation for the LCID and reject negative or 0 numbers.

Reply:
Should I move it to the main path?